sql/pgmoneta_ext--0.1.0.sql

```diff
@@ -1,3 +1,10 @@
 CREATE FUNCTION pgmoneta_ext_version() RETURNS text 
 AS 'MODULE_PATHNAME' 
+LANGUAGE C STRICT;
+
+CREATE FUNCTION pgmoneta_ext_switch_wal(OUT success text,
```

Comment:
`success` isn't `text`

Reply:
I tested the boolean type before. When using the boolean type, it only shows 'f' or 't', not 'false' or 'true'. So, I changed it to text.

<img width="420" alt="Screenshot 2024-05-31 at 6 44 36 AM" src="https://github.com/pgmoneta/pgmoneta_ext/assets/113565433/c02ca84a-63a1-4343-9939-e7582309495f">

Reply:
Yeah, but it is a `bool`

Reply:
Okay, updated.